google/cloud/spanner_v1/client.py

```diff
@@ -289,6 +289,7 @@ def instance(
         configuration_name=None,
         display_name=None,
         node_count=DEFAULT_NODE_COUNT,
```

Comment:
My understanding is that `node_count` and `processing_units` are exclusive (i.e. only one can be specified) so I would change this to `node_count=None` and add logic to set the default if neither are set and raise an error if both are set.